app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -81,21 +81,23 @@ export function EventSelector({
   onEventSelection,
   eventList = WCA_EVENT_IDS,
   disabled = false,
+  maxEvents = WCA_EVENT_IDS.length,
 }) {
   return (
     <>
       <label htmlFor="events">
         {`${I18n.t('competitions.competition_form.events')}`}
         <br />
-        <Button primary type="button" size="mini" id="select-all-events" onClick={() => onEventSelection({ type: 'select_all_events' })}>{I18n.t('competitions.index.all_events')}</Button>
+        <Button disabled={disabled || selectedEvents.length >= maxEvents} primary type="button" size="mini" id="select-all-events" onClick={() => onEventSelection({ type: 'select_all_events' })}>{I18n.t('competitions.index.all_events')}</Button>
```

Comment:
I'm a little confused why this file in the `CompetitionsOverview` has a translation from the `competition_form`.

Reply:
I think this component was just moved to the competition form folder